mkdocs.yml

```diff
@@ -17,6 +17,21 @@ nav:
       - Accessing OmixAtlas: OmixAtlas/Accessing OmixAtlas.md
       - Applications: OmixAtlas/Applications.md
       - Polly Python: OmixAtlas/Polly Python.md
+  - Polly Python: Polly Python/About PollyPy.md
+      - Capabilities Diagram: Polly Python/Capabilities Diagram.md
+      - Installation and Authentication: Polly Python/Installation and Authentication.md
+      - Querying using SQL: Polly Python/Queries/About Queries.md
+          - Basic query examples: Polly Python/Queries/Basic query examples.md
+          - Advanced Query Examples: Polly Python/Queries/Advanced Query Examples.md
+          - Source/Datatype specific queries: Polly Python/Queries/Specific queries.md
+          - Ontology Recommendations: Polly Python/Queries/Ontology Recommendations.md
```

Comment:
Add this in the advanced query section itself

Reply:
Shifted those 2 to the top. They are just for usage definition of source specific queries and recommend(), since that's different from regular sql usage